Makefile

```diff
@@ -0,0 +1,9 @@
+GARBAGE = build cov AUTHORS ChangeLog .eggs lib/pbench.egg-info/ /var/tmp/${USER}
+
+all:
+	ls -ld ${GARBAGE}
+	utils/detox . -print
+
+clean:
+	rm -rf ${GARBAGE}
+	utils/detox .
```

Comment:
These targets are a little weird.

Normally `clean` removes targets and intermediates which were _created by this Makefile_....  And, normally, `make all` does something more than print stuff out....

Is there any particular reason why these two function were written in "make" instead of in, say, bash?

Is this "tool" of sufficient general use to be included in the repo?  And, do we really want it at the _top level_?  (Shouldn't it be in `utils` or similar?)

Reply:
I can take it or leave it, but found it very useful to cleanup the tree.